simulator/virtual_machine.go

```diff
@@ -269,6 +269,20 @@ func (vm *VirtualMachine) apply(spec *types.VirtualMachineConfigSpec) {
 			continue
 		}
 		changes = append(changes, types.PropertyChange{Name: key, Val: val.Value})
+
+		switch key {
+		case "guest.ipAddress":
+			ip := val.Value.(string)
+			vm.Guest.Net[0].IpAddress = []string{ip}
```

Comment:
Hi @dougm,

Looks good. Was hoping to see `if key =~ /guest.net.(.+)/` and translate anything there to `guest.net[0].$1`. Will check if this works as is with IP address alone.

Reply:
We can still do that if needed.